index.js

```diff
@@ -0,0 +1,55 @@
+const moles = document.querySelectorAll('.mole');
+const holes = document.querySelectorAll('.hole');
+const scoreTable = document.getElementById('score');
+const button = document.getElementById('button');
+const game = document.getElementById('game');
+let score = 0;
+let isTimeOver = false;
+let currMole;
+
+const chouseMole = (moles) => {
+  const index = Math.floor(Math.random() * moles.length);
+  const mole = moles[index];
+  if (mole === currMole) {
+    return chouseMole(moles);
+  }
+  currMole = mole;
+  return mole;
+};
+
+const showMole = () => {
+  const DISPLAY_TIME = 700;
+  const mole = chouseMole(moles);
+  mole.classList.add('up');
+  setTimeout(() => {
+    mole.classList.remove('up');
+    if (!isTimeOver) showMole();
+  }, DISPLAY_TIME);
+};
+
+const startGame = () => {
+  score = 0;
+  scoreTable.innerText = score;
+  button.disabled = true;
+  isTimeOver = false;
+  showMole();
+  setTimeout(() => {
+    button.disabled = false;
```

Comment:
If you disable a button, also take care of its css-styles. In this case  at least remove "cursor: pointer". Maybe also consider adding some disabled-status, so that user understands that he can't click this button now. And also think about adding at least "cursor: pointer" to moles, as they are clickable during the game. Don't forget about clickable elements and their styles.